README.md

```diff
@@ -129,6 +132,93 @@ const cell = new BemCell({
 cell.id; // ➜ "button__text@desktop.css"
 ```
 
+### toString()
+
+Returns string representing this cell.
+
+```js
+const BemCell = require('@bem/cell');
+const BemEntityName = require('@bem/entity-name');
+const cell = new BemCell({
+    entity: new BemEntityName({ block: 'button', mod: 'focused' })
+});
+
+cell.toString(); // button_focused
+```
+
+### valueOf()
+
+Returns object representing this cell.
+
+```js
+const BemCell = require('@bem/cell');
+const BemEntityName = require('@bem/entity-name');
+const cell = new BemCell({
+    entity: new BemEntityName({ block: 'button', mod: 'focused' })
+});
+
+cell.valueOf();
+
+// ➜ { entity: { block: 'button', mod: { name: 'focused', value: true } } }
+```
+
+### toJSON()
+
+Returns object for `JSON.stringify()` purposes.
```

Comment:
Returns an object

and not sure about `purposes`, shouldn't it be in singular?